src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -117,6 +121,15 @@ class TriggerView extends Component<TriggerViewProps, TriggerViewState> {
         this.getHostURLConfig()
         this.getWorkflows()
         this.getEnvironments()
+        this.getConfigs()
```

Comment:
this should be inside the if condition

Reply:
Function was not called in if condition.